ee/modules/160-multitenancy-manager/hooks/apis/deckhouse.io/v1alpha1/project_types.go

```diff
@@ -0,0 +1,64 @@
+/*
+Copyright 2023 Flant JSC
+Licensed under the Deckhouse Platform Enterprise Edition (EE) license. See https://github.com/deckhouse/deckhouse/blob/main/ee/LICENSE
+*/
+
+// +kubebuilder:object:generate=true
+// +kubebuilder:validation:Required
+// +groupName=deckhouse.io
+// +versionName=v1alpha1
+
+package v1alpha1
+
+import (
+	apiext "k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+type ProjectSpec struct {
+	// Description of the Project
+	Description string `json:"description,omitempty"`
+
+	// Name of ProjectType to use to create Project
+	ProjectTypeName string `json:"projectTypeName,omitempty"`
+
+	// Values for resource templates from ProjectType
+	// in helm values format that map to the open-api specification
+	// from the openAPI ProjectType field
+	Template map[string]*apiext.JSON `json:"template,omitempty"`
```

Comment:
Could we change a type here to `map[string]interface{}` to avoid unnecessary conventions?

Reply:
We are using `map[string]string` type in log shipper, probably we can do it with map[string]interface{}?
https://github.com/deckhouse/deckhouse/blob/35fc0e30b4b2a17485d1e9ff61a9b5dc3c0d6261/modules/460-log-shipper/apis/v1alpha1/cluster_log_destination.go#L62-L63